spec/ruby/core/process/initialize_spec.rb

```diff
@@ -0,0 +1,31 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe 'Process::Status#initialize' do
+  it 'initializes a new Process::Status instance' do
+    Process::Status.new.should be_an_instance_of(Process::Status)
+  end
+
+  it 'initializes Process::Status with a PID' do
```

Comment:
I have a question regarding these specs. Initially I wrote them as following:

```ruby
describe 'Process::Status#initialize' do
  describe 'with a PID' do
    it 'returns the PID when calling #pid' do
      Process::Status.new(42).pid.should == 42
    end
  end

  describe 'with a PID and exit status' do
    before do
      @status = Process::Status.new(42, 1)
    end

    it 'returns the PID when calling #pid' do
      @status.pid.should == 42
    end

    it 'returns the exit status when calling #exitstatus' do
      @status.exitstatus.should == 1
    end
  end

  # ...
end
```

While this breaks the specs up more nicely and make sure _all_ attributes are set, I can't help but feel that they're getting a bit verbose. Perhaps shared examples would be a solution to this?